### PR TITLE
Whitelist iframe src urls

### PR DESCRIPTION
**Pull request by ryan-verys**
**User Story:**
To allow nodebb admins to control which hostnames are allowed to be passed as src attributes in an embedded iframe. 

This adds the ability to set the allowed Iframe hostnames to pass through with the [newly merged PR](https://github.com/punkave/sanitize-html/pull/192) into the `sanitize-html` module. I've updated `package.json` to reflect the appropriate version number from NPM (1.17.0). 

Users can now pass a `allowedIframeHostnames` property with an array value as part of the option object that gets passed to the `sanitizeHtml` function. The settings for this array can now be controlled with this PR.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var	sanitizeHtml = require('sanitize-html'),
+var	sanitizeHtml = require('/Users/ryan/Verys/sanitize-html'),
```
Comment (akhoury) [inline index.js]: that looks like a local module, that won't work

Comment (ryan-verys) [reply index.js]: Doh! Updated :).

**Comment by ryan-verys** [comment]
All set.

**Comment by akhoury** [comment]
`nodebb-plugin-sanitizehtml@0.2.7` published

**Comment by akhoury** [comment]
thanks @ryan-verys

**Pull request merged**